packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/getLastZIndex.ts

```diff
@@ -0,0 +1,22 @@
+import { getTagOfNode } from 'roosterjs-editor-dom';
+
+/**
```

Comment:
Please add "@internal" in comment

I need to check why  the build tool doesn't fail since "@internal" is required here.